network-libp2p/src/config.rs

```diff
@@ -58,7 +62,7 @@ impl Config {
                 let mut s = DefaultHasher::new();
                 message.topic.hash(&mut s);
                 message.data.hash(&mut s);
-                gossipsub::MessageId::from(s.finish().to_string())
+                gossipsub::MessageId::from(s.finish().to_be_bytes())
```

Comment:
Unrelated to this PR: `DefaultHasher` is not guaranteed to stay the same across Rust versions. Is it important that this is the same for all of the network? Couldn't immediately see that from the libp2p-gossipsub documentation: [`message_id_fn`](https://docs.rs/libp2p-gossipsub/0.46.1/libp2p_gossipsub/struct.ConfigBuilder.html#method.message_id_fn).

Reply:
Mmm, I'd say it is important. `libp2p-gossipsub` documentation is not very clear on this. But then, how would you propose to go since we can't use the inner `SipHashers` as they are deprecated

Reply:
We could use a standard cryptographic hash function, I guess.

Reply:
I just changed it to use `sha256`

Reply:
Nice. :)